library/core/functions.general.php

```diff
@@ -3565,6 +3565,26 @@ function trustedDomains() {
     }
 }
 
+if (!function_exists('isTrustedUrl')) {
```

Comment:
Oh thanks. I'll extend isTrustedDomain to support what I need. Longer term we'll need to make a class.